extensions/ray-so/src/create-a-snippet.tsx

```diff
@@ -59,19 +69,43 @@ export default function CreateSnippet() {
         onChange={(snippet) => setCode({ ...code, snippet })}
       />
       <Form.Separator />
-      <Form.Dropdown id="color" title="Color" storeValue onChange={(color) => setCode({ ...code, color })}>
-        {colors.map((el, idx) => (
-          <Form.Dropdown.Item
-            key={idx}
-            icon={{ source: `${el}.png` }}
-            value={el}
-            title={el.charAt(0).toUpperCase() + el.substring(1).toLowerCase()}
-          />
-        ))}
+      <Form.Dropdown
+        id="color"
+        title="Color"
+        storeValue
+        defaultValue={defaultSnippet.color}
+        onChange={(color) => setCode({ ...code, color })}
+      >
+        <Form.Dropdown.Section title="Partners">
+          {data?.themes.map(
+            (el: { id: string; name: string; partner?: boolean }, idx: number) =>
+              el.partner && (
+                <Form.Dropdown.Item
+                  key={idx}
+                  icon={{ source: `${el.id}.svg`, tintColor: Color.PrimaryText }}
```

Comment:
I realized this import will fail whenever we add new partner themes, so I added an ﻿`iconUrl` field to the API response that can be used instead. This way, we don't need to keep any logos in this repository, and everything will stay up-to-date automatically :)

Reply:
Just made the change! Although I'm not sure if I need to do something else with the SVGs once they're served by the Config API, since I'm seeing them rather small in the Dropdown Items:

<img width="529" alt="image" src="https://github.com/raycast/extensions/assets/64607766/cfee5c00-2cd0-4ae1-b33c-b30ba08661d8">

It may be just on my end, so if you can confirm there's nothing more to be done, I think it should be ready!

Reply:
My bad, the svg:s needed to be sized correctly for this to work so updated it on the API side.